snapshot/git/gitfiler/checkouter.go

```diff
@@ -82,9 +83,11 @@ func (c *Checkouter) CheckoutAt(id string, dir string) (co snapshot.Checkout, er
 	return &UnmanagedCheckout{id: id, dir: dir}, nil
 }
 
-// Implement noop ingest so this Checkouter can be passed around as a Filer.
+// Implement noop ingest/update so this Checkouter can be passed around as a Filer.
```

Comment:
Since we're editing this code, it would be nice to have a comment with the Checkouter type declaration explaining that Checkouter is a Filer that only implements checkout from a pool of repos (managed by RepoPool).